### PR TITLE
Export recorded runs

### PR DESCRIPTION
**Pull request by craig-willis**
Fixes https://github.com/whole-tale/whole-tale/issues/90

This PR:
* Updates `manifest.py` to include recorded run information and files if one or more runs are present for a version
* Updates exported tales to include updated manifest.json and run files in payload

To test:
* Create a new tale with two entrypoint scripts that create output files
* Execute a recorded run for each entrypoint
* Export tale/version associated with the run
* Confirm that the `manifest.json` includes information for both runs 
* Confirm that payload includes files from both runs

### CODE REVIEW

File: server/lib/exporters/__init__.py [hunk 1]
```diff
@@ -131,28 +146,26 @@ def append_aggergate_checksums(self):
         """
         aggs = self.manifest["aggregates"]
         for path, chksum in self.state['md5']:
-            uri = "./" + path.replace("data/", "", 1)
+            uri = "./" + path.replace("data/", "")
```
Comment (Xarthisius) [inline server/lib/exporters/__init__.py:149]: what if there's a `data` folder inside `data` folder, or even `data` inside the data inside the data? People did crazier things...

File: server/lib/exporters/__init__.py [hunk 2]
```diff
@@ -131,28 +146,26 @@ def append_aggergate_checksums(self):
...
-            uri = prepended_path + relpath
+        for fullpath, relpath in self.list_files():
+            uri = "./" + relpath
             index = self._agg_index_by_uri(uri)
             if index is not None:
                 aggs[index]["wt:mimeType"] = (
                     magic_wrapper.from_file(fullpath) or "application/octet-stream"
                 )
-            aggs[index]["wt:size"] = os.path.getsize(fullpath)
+                aggs[index]["wt:size"] = os.path.getsize(fullpath)
```
Comment (Xarthisius) [inline server/lib/exporters/__init__.py:168]: good catch!

File: server/lib/exporters/__init__.py (hunk 1 above)
Comment (craig-willis) [reply server/lib/exporters/__init__.py:149]: Not sure why I dropped that, good catch.

File: circle.yml [hunk 3]
```diff
@@ -2,7 +2,7 @@ version: 2.0
 jobs:
   build:
     docker:
-      - image: wholetale/girder:latest
+      - image: craigwillis/girder:rrevents
```
Comment (Xarthisius) [inline circle.yml:5]: Don't forget to revert it.

**Pull request merged**